sdcm/tester.py

```diff
@@ -2909,6 +2912,7 @@ def tearDown(self):
         self.stop_event_device()
         if self.params.get('collect_logs'):
             self.collect_sct_logs()
+        cleanup_ssl_config()
```

Comment:
you probably want to add @silence ontop of it.

so it never gonna fail the test